app/service/planning_svc.py

```diff
@@ -139,15 +139,24 @@ def __init__(self):
             variables = re.findall(r'#{(.*?)}', decoded_test, flags=re.DOTALL)
             if variables:
                 agent_facts = await self._get_agent_facts(operation['id'], agent['paw'])
+                requires_relationship = await self.get_service('data_svc').dao.get('core_ability_relationships',
```

Comment:
@privateducky should this get a new data_svc function and avoid directly calling the Dao? what's the best practice here?

Reply:
I'm working on adding the fact relationship data to the dictionary returned by explode_operations to alleviate the need to make database queries so many times